go/pvl/helpers.go

```diff
@@ -102,10 +102,12 @@ func pvlJSONStringSimple(object *jsonw.Wrapper) (string, error) {
 }
 
 // pvlSelectionContents gets the HTML contents of all elements in a selection, concatenated by a space.
-func pvlSelectionContents(selection *goquery.Selection, useAttr bool, attr string) (string, error) {
-	len := selection.Length()
-	results := make([]string, len)
-	errs := make([]error, len)
+// If getting the contents/attr value of any elements fails, that does not cause an error.
+// The result can be an empty string.
+func pvlSelectionContents(selection *goquery.Selection, useAttr bool, attr string) string {
+	length := selection.Length()
+	results := make([]string, length)
+	errs := make([]error, length)
```

Comment:
but you don't care about errors. it's just a question of keeping the text around if the `err` is non-`nil`